app/models/institution.rb

```diff
@@ -156,6 +156,7 @@ class Institution < ApplicationRecord
   has_many :institution_programs, -> { order(:description) }, inverse_of: :institution, dependent: :nullify
   has_many :school_certifying_officials, -> { order 'priority, last_name' },
            primary_key: :facility_code, foreign_key: 'facility_code', inverse_of: :institution
+  belongs_to :version
```

Comment:
The migration for the association has been run, but they were never set up in the models.